src/SFA.DAS.ApplyService.InternalApi/Controllers/GatewayReviewController.cs

```diff
@@ -51,11 +52,24 @@ public async Task EvaluateGateway(Guid applicationId, [FromBody] EvaluateGateway
         {
             await _mediator.Send(new EvaluateGatewayRequest(applicationId, request.IsGatewayApproved, request.EvaluatedBy));
         }
+
+        [HttpPost("GatewayReview/{applicationId}/Withdraw")]
+        public async Task<bool> WithdrawApplication(Guid applicationId, [FromBody] GatewayWithdrawApplicationRequest request)
```

Comment:
Does this need to be a FromBody? Doesn't look like it.  Have a look and leave as is, if it does.

Reply:
it contains comments & user info which would be too long via query string